### DISCUSSION TITLE
Is this library abandonware now?

### DISCUSSION THREAD

**Discussion by byF** [Hopding/pdf-lib#1318] [General]
I've read the [MAINTAINERSHIP](https://github.com/Hopding/pdf-lib/blob/master/docs/MAINTAINERSHIP.md) doc, but it seems most of the issues and PRs are very stale now.

Are there any plans for moving this library under different ownership?

**Comment by adamcollingburn** [comment]
It seems like it is no disrespect to the author - maintaining well-used projects like this can consume all your free time and eventually lead to burnout. I was considering switching to this library from our paid PDFLib, but we can't really deal with a breaking change from a node update or something (not likely I know) with no one around to help fix it ASAP, as we rely on this stuff to produce 1000's of print on demand artworks everyday.

**Comment by joewestcott** [comment]
As someone who's submitted multiple PRs to this repo, it's sad to see the community fragment into multiple forks again. So many developer hours are being lost troubleshooting the same bugs over and over again. There's no doubt this project requires a special set of skills and Andrew has done an incredible job with it.

> maintaining well-used projects like this can consume all your free time and eventually lead to burnout.

That's why multiple maintainers are key! That should be the final job of any open source maintainer.

**Comment by byF** [comment]
I agree; I can imagine maintaining a library such as this is quite a burden, so while burnout is understandable, it's sad it all rests on a single person. I don't see other viable options in Node/JS space, I'd like this library to survive.

**Comment by laneme** [comment]
@andrew-dillon @Hopding
I'm not sure if andrew is still up on github but a maintainence notification on the readme would be nice.

Maybe find a well-known contributor/company as a maintainer for now, so in the future they can add maintainers willing to contribute/submit PRs/maintain. 

This lib is awesome <3

**Comment by satyajitnayk** [comment]
@byF @joewestcott @adamcollingburn  There is a fork of this https://github.com/cantoo-scribe/pdf-lib. This is actively maintained as well & I found it useful .
